crates/hcl-edit/Cargo.toml

```diff
@@ -30,14 +30,14 @@ default = []
 perf = ["hcl-primitives/perf"]
 
 [dependencies]
-fnv = "1.0.7"
+fnv = "1.0"
 hcl-primitives = { version = "0.1.2", path = "../hcl-primitives" }
-vecmap-rs = "0.1.12"
-winnow = "0.5.15"
+vecmap-rs = "0.2"
```

Comment:
Be sure to also bump this in `crates/hcl-rs/Cargo.toml`

Reply:
yeah, double checked it and I didn't forget to adjust it 😄